src/main/resources/view/ProfilePanel.fxml

```diff
@@ -9,40 +9,42 @@
 <?import javafx.scene.layout.Pane?>
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.shape.Ellipse?>
-<?import javafx.scene.text.Font?>
-<?import javafx.scene.text.Text?>
 
 <!-- @@author Livian1107 -->
 
-<Pane xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<Pane prefHeight="500.0" prefWidth="731.0" xmlns="http://javafx.com/javafx/8.0.121" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Please help to check that all fxml files are using java version 8 ```http://javafx.com/javafx/8``` before merging, thank you!